build.rs

```diff
@@ -200,6 +204,20 @@ const ASM_TARGETS: &[AsmTarget] = &[
         asm_extension: "S",
         preassemble: false,
     },
+    AsmTarget {
```

Comment:
Let's remove the `AsmTarget` stuff for MIPS since my goal is to not have any assembly code for MIPS.